TheMET/TheMET/Network/MetAPI.swift

```diff
@@ -256,3 +256,8 @@ class MetAPI{
         return result
     }
 }
+
+enum MetAPIError: Error {
+    case metAPIdoesNotAnswer
```

Comment:
1) CamelCase broken
2) According to the implementation this error is thrown when MetAPI object doesn't exist anymore, not when its doesn't answer. Pls rename it.